.pre-commit-config.yaml.jinja

```diff
@@ -0,0 +1,97 @@
+exclude: |
+  (?x)
+  # Files and folders generated by bots, to avoid loops
+  ^setup/|/static/description/index\.html$|
+  # Maybe reactivate this when all README files include prettier ignore tags?
+  ^README\.md$|
+  # Library files can have extraneous formatting (even minimized)
+  /static/(src/)?lib/|
+  # Repos using Sphinx to generate docs don't need prettying
+  ^docs/_templates/.*\.html$|
+  # You don't usually want a bot to modify your legal texts
+  (LICENSE.*|COPYING.*)
```

Comment:
One thing we'll want is a mechanism to exclude uninstallable addons. @ThomasBinsfeld has a pre-commit hook that does that. This can be done in a follow-up.

Reply:
OK, although AFAIK nowadays all new branches start empty, so there should be no uninstallable addons there.